DeviceManager/SerializationModels.py

```diff
@@ -20,6 +20,12 @@ def validate_attr_label(input):
         raise ValidationError("Labels must contain letters, numbers or dashes(-_)")
     return '-- invalid --'
 
+def validate_children_attr_label(attr_label):
+    unique = { each['label'] : each for each in attr_label }.values()     
+    if len(attr_label) > len(unique):
+        raise ValidationError('a template cant not have repeated attributes')
+    return '-- invalid --'    
```

Comment:
Why does it return invalid when the validation succeded?

Reply:
Hi @ericbaum, you're right, actually return instruction is not necessary. I removed the return and tested again and everything worked normally.